release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -1130,7 +1130,7 @@ With this release, when you apply a performance profile, the Node Tuning Operato
 [id="ocp-4-18-support-for-worker-nodes-on-amd-cpus_{context}"]
 ==== Support for worker nodes with AMD EPYC Zen 4 CPUs
 
-With this release, you can use the `PerformanceProfile` custom resource (CR) to configure worker nodes on machines equipped with AMD EPYC Zen 4 CPUs (Genoa and Bergamo). These CPUs are fully supported.
+With this release, you can use the PerformanceProfile custom resource (CR) to configure worker nodes on machines equipped with AMD EPYC Zen 4 CPUs (Genoa and Bergamo). These CPUs are fully supported when configured with a single NUMA domain (NPS=1).
```

Comment:
```suggestion
With this release, you can use the `PerformanceProfile` custom resource (CR) to configure worker nodes on machines equipped with AMD EPYC Zen 4 CPUs (Genoa and Bergamo). These CPUs are fully supported when configured with a single NUMA domain (NPS=1).
```

Tiny nit to add back ticks as this is an [object name](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references). 

> An object reference is when you are referring to the actual instance of an API object, where the object name is important.